config/tileset/aerial.json

```diff
@@ -39,7 +39,6 @@
     { "2193": "im_01F6P1V84P2HJ1YB0D5X1CSWJH", "3857": "im_01ED83DC7MCMBCRNN45E07HCHH", "name": "tasman_rural_2016-17_0-3m", "minZoom": 13 },
     { "2193": "im_01FB82NZY0ASDQBKT28DSXS8XC", "3857": "im_01FB82QKPWEHV0FWN7KBXM6NDM", "name": "wellington_rural_2021_0-3m_RGB", "minZoom": 13 },
     { "2193": "im_01F6P220GYWC836YQ4RRZGSV73", "3857": "im_01ED83VT2T06B1FVXF9P6CCV8C", "name": "west-coast_rural_2016-17_0-3m", "minZoom": 13 },
-    { "2193": "im_01F6P15FS34RD0KX1FQCQF1CA8", "3857": "im_01ECTS06BSSHA626JHSDTNSC97", "name": "gisborne_rural_2017-18_0-3m", "minZoom": 13 },
```

Comment:
Is this one removed on purpose?

Reply:
there is two commits that reference this, maybe i should cherry pick them into their own PR